### DISCUSSION TITLE
APK Editor Studio v1.7.1

### DISCUSSION THREAD

**Discussion by kefir500** [kefir500/apk-editor-studio#106] [Releases]
- Added support for optimizing shared libraries.
- Added Arabic translation.
- Added Spanish (Mexican) translation.
- Fixed Apktool update loop when Java is not found.
- Windows: Improved Java path detection.

<hr /><em>This discussion was created from the release <a href='https://github.com/kefir500/apk-editor-studio/releases/tag/v1.7.1'>APK Editor Studio v1.7.1</a>.</em>

**Comment by harmit03** [comment]
After installing and running the app for the first time it prompts that mspc140.dll not found. After I somehow put the required dll file to the installation folder, it prompts that app cannot start due to error. Any solutions?

version 1.2.0 runs fine but after that every update is giving the same error.

I also tried reinstalling ms visual c++ but that too doesn't work.

**Comment by kefir500** [comment]
@harmit03, Have you tried the Visual C++ Redistributable from [this link](https://aka.ms/vs/16/release/vc_redist.x86.exe)?

**Comment by harmit03** [comment]
It worked, thank you very much for your guidance, I only had x64 2019
installed on my system.

once again thank you very much for such a useful tool

good day.

**Comment by Output977** [comment]
APK Editor签名怎么支持PKCS12类型密钥呢，我用android studio生成的签名密钥是PKCS12，在APK Editor平台读取不到该密钥
